build/log.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/btcsuite/btclog"
 )
 
+var (
+	// RootLogWriter is the main parent log writer all sub loggers should be
+	// appended to.
+	RootLogWriter = NewRotatingLogWriter()
```

Comment:
if this is a global variable the multiple importers of the `build` package will overwrite each others logs. for example, down the road when we move to above in-process itests, multiple lnds would all be using the same log instance. there are some loose plans to eventually remove all global log variables from our packages, so this would be a step in the opposite direction :P

Reply:
Ah yes, not a good idea then. Moved it back to the main package. How do we plan to remove the global log variables? Sounds like a pretty big refactor.